src/Microsoft.ML.Data/EntryPoints/InputBuilder.cs

```diff
@@ -791,7 +791,7 @@ public static class Range
         /// </summary>
         public static class Deprecated
         {
-            public new static string ToString() => "Deprecated";
+            new public static string ToString() => "Deprecated";
```

Comment:
😕 This is super weird. How is it used?

Reply:
It is very strange. So, there is some scheme for defining an "entry-point graph" which is a structure defined as a JSON graph which can be serialized and sent across the wire. So there is this utility class here for holding the standard JSON property names... that is fine, so far as it goes, assuming we accept the idea of entry-points being structured in this incredibly weakly typed fashion. But let's set that aside. Now then we *could* have had a property named `JsonPropertyName` or something on this, why the devil would we among all possible names choose `ToString` specifically, which requires this odd hiding? That seems like at best a poor choice.

I'd prefer though not to deal with these odd things in this PR.